pkgs/applications/science/electronics/pulseview/default.nix

```diff
@@ -1,45 +1,33 @@
-{ lib, stdenv, fetchurl, fetchpatch, pkg-config, cmake, glib, boost, libsigrok
-, libsigrokdecode, libserialport, libzip, udev, libusb1, libftdi1, glibmm
-, pcre, python3, qtsvg, qttools, wrapQtAppsHook, desktopToDarwinBundle
+{ lib, stdenv, fetchgit, pkg-config, cmake, glib, boost, libsigrok
+, libsigrokdecode, libserialport, libzip, libftdi1, hidapi, glibmm
+, pcre, python3, qtsvg, qttools, bluez
+, wrapQtAppsHook, desktopToDarwinBundle
 }:
 
 stdenv.mkDerivation rec {
   pname = "pulseview";
-  version = "0.4.2";
+  version = "nightly-2023-10-21";
```

Comment:
Hi!

I've got the same suggestion [two months ago](https://github.com/NixOS/nixpkgs/pull/259629#discussion_r1349656840).

Since upstream preferred nightly, I chose nightly.
But if this is what stops this from getting merged, I'll change it to unstable.

Reply:
> But if this is what stops this from getting merged, I'll change it to unstable.

Just a matter of preference I guess. In any case, I can't merge anyway.

Reply:
I much prefer `unstable` over `nightly`. It may be that upstream calls them `nightly` because they rebuild them every day (or night), but here at nixpkgs we won't do this and instead will just bump the version every now and then (every couple of weeks I guess).

Reply:
Majority wins - I updated it to unstable vocabulary (and updated it to the latest commits available now)